packages/utils/index.ts

```diff
@@ -311,7 +311,13 @@ export const formatNameWithRules = (
   });
   return result;
 };
-
+const DEFAULT_USER_ID = '00000000-0000-0000-0000-000000000000';
+export function isValidUserId(id?: string): boolean {
+  if (!id) {
+    return false;
```

Comment:
return  id && id !==DEFAULT_USER_ID is beter

Reply:
next time， i will change